sts_auth/stsauth.py

```diff
@@ -93,39 +110,37 @@ def parse_config_file(self):
         """Read configuration file and only set values if they
         were not passed in from the CLI.
         """
-        if self.config.has_section('default'):
-            logger.debug('Found \'default\' section in'
-                         ' {0.credentialsfile!r}!'.format(self))
-            default = self.config['default']
-            msg = ('Attribute {1!r} not set, using value from {0.credentialsfile!r}')
+        if self.config.has_section("default"):
+            logger.debug("Found 'default' section in" " {0.credentialsfile!r}!".format(self))
```

Comment:
Changes like these should be merged into one string if they're moved to a single.

Reply:
@phillipjf fixed